src/packagedcode/windows_helpers.py

```diff
@@ -0,0 +1,311 @@
+import os
+from pathlib import PureWindowsPath
+
+import attr
+import logbook
```

Comment:
Let's not use a new logging lib for now.

Reply:
I've removed the import to logbook, but it will still be installed as a dependency for regipy